components/emma-pricing-cards/components/PricingCard.js

```
@@ -0,0 +1,96 @@
+      <h3 className="text-xl font-bold text-gray-900 dark:text-white mb-2">
+        {display_name__limio}
+      </h3>
+
+      <div className="mb-4">
+        {isContactUs ? (
+          <span className="text-2xl font-extrabold text-gray-900 dark:text-white">Contact us</span>
+        ) : (
+          <div className="flex items-baseline">
+            <span
+              className="text-3xl font-extrabold text-gray-900 dark:text-white"
+              dangerouslySetInnerHTML={{
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)